### ISSUE TITLE
The results based on S3DIS dataset

### ISSUE THREAD

**Issue by LeeLinSCUT** [Yang7879/3D-BoNet#37]
I've trained the module through 50 epochs and then finished the evlauation on Area_5. But later I've noticed the mPre and mRec is about 51 and 39, which is different from the Table 3.  I haven't remove any loss or branch. Could you tell me the reason? Is the released module trained on more epochs?

**Comment by Yang7879** [comment]
Hi @LeeLinSCUT, the hyperparameters are the same as what I used to train, but the results may indeed be different (can be higher or lower), probably because the association algorithm is not always stable enough.
